app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt

```diff
@@ -64,6 +64,7 @@ open class MainPresenterImpl(val mapzenLocation: MapzenLocation, val bus: Bus,
   override var resultListVisible = false
   override var reverseGeo = false
   override var reverseGeoLngLat: LngLat? = null
+  override var currentSearchIndex: Int = 0
```

Comment:
This flag should be reset to 0 when the current set of search results is cleared.

Otherwise the next time a search is executed the current index from the previous set of results is displayed by default.